model/articles.models.js

```diff
@@ -11,4 +11,16 @@ function fetchArticleById(article_id) {
   });
 }
 
-module.exports = { fetchArticleById };
+function fetchArticle(sort_by = "created_at", order = "desc") {
```

Comment:
are you fetching one article or multiple articles? is there a better function name than "fetchArticle"?